scripts/appleseed.package.py

```diff
@@ -29,8 +29,7 @@
 #
 
 from __future__ import print_function
-from distutils import archive_util, dir_util
-from xml.etree.ElementTree import ElementTree
+
```

Comment:
What's the rationale behind moving these imports around?

Reply:
No rationale. I did a code format in PyCharm and it moved it

Reply:
Doesn't make much sense to separate them like that :) Let's revert this. Feel free to merge the PR afterward (have you got the rights to do so?)

Reply:
I’m pretty sure I do